src/main/resources/application.yml

```diff
@@ -38,6 +38,14 @@ folio:
   tenant:
     validation:
       enabled: true
+  kafka:
+    numberOfPartitions: ${NUMBER_OF_PARTITIONS:1}
+    replicationFactor: ${REPLICATION_FACTOR:1}
+    listener:
+      check-in:
+        concurrency: ${KAFKA_EVENTS_CONCURRENCY:5}
+        topic-pattern: ${KAFKA_EVENTS_CONSUMER_PATTERN:(${folio.environment}\.)[a-zA-z0-9-]+\.\w+\.check-in}
```

Comment:
Do we want to listen to only dcb enabled tenants ? As per this it will listen to all the check-in events that are published. I dont think that is a good idea.

Reply:
fixed